earth_enterprise/src/fusion/autoingest/storage/CombinedRPConfig.idl

```diff
@@ -19,6 +19,7 @@
 #include <autoingest/.idl/storage/MaskgenConfig.h>
 #include <autoingest/.idl/storage/MosaicConfig.h>
 #include <autoingest/.idl/storage/KRPConfig.h>
+#include "CacheSizeCalculations.h"
```

Comment:
Minor: you could save yourself a little pain by adding this line as part of the autogen code as well.